activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeEvents.java

```diff
@@ -112,4 +117,105 @@ public void shouldGetJustThreeVariableCreatedEvent(){
                 .isNotEmpty()
                 .hasSize(3);
     }
+    
+    @Test
+    public void shouldGetSignalReceivedEventsForProcessWithSignalStart(){
+
+    	//In this test processInstanceId is null
+        //given
+        securityUtil.logInAs("salaboy");
+
+        //when
+        ProcessInstance categorizeProcess = processRuntime.start(ProcessPayloadBuilder.start()
```

Comment:
In this case you should not start an instance of the process, as the instance will be created when the signal is triggered. Your test should still work if you remove `processRuntime.start`.